### DISCUSSION TITLE
c++, compile aot failed on windows, "could not open 'libcmt.lib'"

### DISCUSSION THREAD

**Discussion by ActivePeter** [WasmEdge/WasmEdge#2033] [Q&A]
![image](https://user-images.githubusercontent.com/8100696/198985861-16f875ad-feae-404a-ba16-be09e4d46e71.png)
The guide I followed to：[https://wasmedge.org/book/en/sdk/c.html](https://wasmedge.org/book/en/sdk/c.html)
My Repository：[https://github.com/ActivePeter/pa_code_playground/tree/main/wasm/wasm_edge_test](https://github.com/ActivePeter/pa_code_playground/tree/main/wasm/wasm_edge_test)

**Comment by hydai** [comment]
I think it is related to this issue: https://github.com/WasmEdge/WasmEdge/issues/1898

Not sure about the working envs on the windows platform.
